stdlib/Distributed/test/splitrange.jl

```diff
@@ -22,7 +22,7 @@ using Distributed: splitrange
 @test splitrange(-1, 1, 4) == Array{UnitRange{Int64},1}([-1:-1,0:0,1:1])
 
 const BASE_TEST_PATH = joinpath(Sys.BINDIR, "..", "share", "julia", "test")
-isdefined(Main, :OffsetArrays) || @eval Main include(joinpath($(BASE_TEST_PATH), "testhelpers", "OffsetArrays.jl"))
+isdefined(Main, :OffsetArrays) || @eval Main @everywhere include(joinpath($(BASE_TEST_PATH), "testhelpers", "OffsetArrays.jl"))
```

Comment:
What's the reason for this change? Could this cause problems in tests running in other workers? What if it defines methods that are supposed to fail in those tests?

Reply:
I'm unsure about this, though.

Without this `Distributed` test is broken:

```
      From worker 10:	ERROR: LoadError: LoadError: TaskFailedException
      From worker 10:	
      From worker 10:	    nested task error: On worker 25:
      From worker 10:	    UndefVarError: OffsetArrays not defined
      From worker 10:	    Stacktrace:
      From worker 10:	      [1] deserialize_module
      From worker 10:	        @ /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.6/Serialization/src/Serialization.jl:957
      From worker 10:	      [2] handle_deserialize
      From worker 10:	        @ /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.6/Serialization/src/Serialization.jl:856
      From worker 10:	      [3] deserialize
      From worker 10:	        @ /buildworker/worker/package_linux32/build/usr/share/julia/stdlib/v1.6/Serialization/src/Serialization.jl:774
```

Adding this helps remove this error.

Reply:
Maybe https://github.com/JuliaLang/julia/blob/e37876793f77b516c30000b66b373e71b1e462e5/test/offsetarray.jl#L4

Reply:
There's already `using .Main.OffsetArrays` one line down below on this.

I think it's because `eachindex` now requires `OffsetArrays.IdOffsetRange` and triggers the error, while for `OffsetArrays` it uses `IdentityUnitRange` and doesn't require anything in OffsetArrays.